cmd/chart/verify.go

```diff
@@ -44,6 +45,7 @@ autoscaler          ghcr.io/openfaasltd/autoscaler:0.2.5
 		}
 
 		verbose, _ := cmd.Flags().GetBool("verbose")
+		maxLevel, _ := cmd.Flags().GetInt("max")
```

Comment:
Please validate the flag in PreRunE

Can we call "max" "depth" instead?

Reply:
Done